python/lsst/daf/butler/script/ingest_zip.py

```diff
@@ -46,7 +48,14 @@ def ingest_zip(
         URI string of the location of the Zip file.
     transfer : `str`, optional
         Transfer mode to use for ingest.
+    transfer_dimensions : `bool`
+        Indicate whether dimensions should be transferred along with
+        datasets. It can be more efficient to disable this if it is known
+        that all dimensions exist.
```

Comment:
But default is already `False`? Or is default for parameter different from CLI default?

Reply:
The script interface gets called with every parameter from the CLI. I will clarify the text.

Reply:
I realized that I didn't really want the CLI default to be True so I just fixed that (people should be intentional about asking for dimension records to be inserted)